packages/repository/src/repositories/repository.ts

```diff
@@ -19,7 +19,9 @@ import {Filter, Where} from '../query';
 
 // tslint:disable:no-unused-variable
 
-export interface Repository<T extends Model> {
+export interface Repository<T extends Model> {}
+
+export interface ExecutableRepository<T extends Model> extends Repository<T> {
```

Comment:
The git compare is weird, it should show the entire new interface instead of the declaration part only.
@shimks Do we still need the `execute` function in https://github.com/strongloop/loopback-next/blob/14a312c91639769111c829b07b37fa9e13b4bf4f/packages/repository/src/repositories/repository.ts#L333?

Reply:
According to @bajtos, `DefaultCrudRepository` should extend from an executable repository, so I guess we still do :p